typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -82,7 +82,7 @@ export enum WarpRouteIds {
   // TODO: uncomment after merging the staging route to registry
   // this has been commented out as it leads to check-warp-deploy cron job failing
   // SuperTokenStaging = 'USDTSTAGING/base-bitlayer-celo-ethereum-fraxtal-ink-linea-lisk-mantle-metal-metis-mode-optimism-ronin-soneium-sonic-superseed-unichain-worldchain',
-  SuperUSDT = 'USDT/base-bitlayer-celo-ethereum-fraxtal-ink-linea-lisk-mantle-metal-metis-mode-optimism-ronin-soneium-sonic-superseed-unichain-worldchain',
+  SuperUSDT = 'oUSDT/base-bitlayer-celo-ethereum-fraxtal-ink-linea-lisk-mantle-metal-metis-mode-optimism-ronin-soneium-sonic-superseed-unichain-worldchain',
```

Comment:
might be a little annoying, but would you mind renaming all the `supertoken` references to just `ousdt` and `ousdtstaging` please 🙏